boefjes/boefjes/models.py

```diff
@@ -34,6 +47,7 @@ class Boefje(Plugin):
     boefje_schema: dict | None = None
     cron: str | None = None
     interval: int | None = None
+    run_on: list[RunOn] | None = None  # {"run_on": "create_update"} -> {"run_on": ["create", "update"]}
```

Comment:
IMO the type suggestion is already clear on how to use this

```suggestion
    run_on: list[RunOn] | None = None
```